packages/code-du-travail-modeles/src/modeles/conventions/573_commerces_de_gros/indemnite-licenciement.yaml

```
@@ -0,0 +1,343 @@
+  applicable si: catégorie professionnelle = 'Cadres'
+
+contrat salarié . convention collective . commerces de gros . catégorie professionnelle . cadres . cadre durant au moins de 15 ans:
+  titre: Le salarié a été cadre durant au moins 15 ans
+  question: Le salarié a-t-il été cadre durant au moins 15 ans ?
+  cdtn:
+    type: oui-non
+
+contrat salarié . convention collective . commerces de gros . catégorie professionnelle . cadres . cadre durant au moins de 15 ans . oui:
+  applicable si: cadre durant au moins de 15 ans = 'Oui'
+
+contrat salarié . convention collective . commerces de gros . catégorie professionnelle . cadres . cadre durant au moins de 15 ans . oui . age:
+  titre: Âge du salarié
+  question: Quel est l'âge du salarié ?
+  cdtn:
+    type: entier
+
+contrat salarié . convention collective . commerces de gros . catégorie professionnelle . agents:
+  applicable si: catégorie professionnelle = 'Agents de maîtrise, techniciens et assimilés'
+
+contrat salarié . convention collective . commerces de gros . catégorie professionnelle . agents . licenciement économique:
+  titre: Licenciement pour motif économique
+  question: Le motif du licenciement est-il économique ?
+  cdtn:
+    type: oui-non
+
+contrat salarié . convention collective . commerces de gros . catégorie professionnelle . agents . licenciement économique . oui:
+  applicable si: licenciement économique = 'Oui'
+
+contrat salarié . convention collective . commerces de gros . catégorie professionnelle . agents . licenciement économique . oui . age:
+  titre: Âge du salarié
+  question: Quel est l'âge du salarié ?
+  cdtn:
+    type: entier
+
+contrat salarié . convention collective . commerces de gros . catégorie professionnelle . autres:
+  applicable si: catégorie professionnelle = 'Autres salariés'
+
+# Formules
+contrat salarié . convention collective . commerces de gros . au delà de dix ans:
+  somme:
+    - contrat salarié . indemnité de licenciement . ancienneté conventionnelle en année
+    - (- 10 an)
+  plancher: 0 an
+
+contrat salarié . convention collective . commerces de gros . de zero à neuf ans:
+  valeur: contrat salarié . indemnité de licenciement . ancienneté conventionnelle en année
+  plafond: 9 an
+  plancher: 0 an
+
+contrat salarié . convention collective . commerces de gros . au delà de neuf ans:
+  somme:
+    - contrat salarié . indemnité de licenciement . ancienneté conventionnelle en année
+    - (- 9 an)
+  plancher: 0 an
+
+contrat salarié . convention collective . commerces de gros . de dix à dix-neuf ans:
+  valeur: au delà de neuf ans
+  plafond: 10 an
+  plancher: 0 an
+
+contrat salarié . convention collective . commerces de gros . au delà de vingt ans:
+  somme:
+    - contrat salarié . indemnité de licenciement . ancienneté conventionnelle en année
+    - (- 19 an)
+  plancher: 0 an
+
+contrat salarié . convention collective . commerces de gros . formule 1 autre:
```

Pourquoi ne pas mettre ces formules dans le namespace de la catégorie autre ?

```suggestion
contrat salarié . convention collective . commerces de gros . catégorie professionnelle . autres . formule 1:
```

ça correspond plus à la logique namespace du coup ?

D'ailleurs, formule 1 autre et formule 2 autre n'étant pas réutilisé, on peut les fusionner dans somme autre:

```yaml
contrat salarié . convention collective . commerces de gros . catégorie professionnelle . autres . somme:
  somme:
    - produit:
        assiette: contrat salarié . indemnité de licenciement . salaire de référence conventionnel * contrat salarié . indemnité de licenciement . ancienneté conventionnelle en année
        facteur: 1 / 5
    - produit:
       assiette: contrat salarié . indemnité de licenciement . salaire de référence conventionnel * au delà de dix ans
       facteur: 2 / 15
  unité: €
```